src/node_crypto.cc

```diff
@@ -2097,6 +2099,38 @@ void SSLWrap<Base>::GetPeerCertificate(
 }
 
 
+template <class Base>
+void SSLWrap<Base>::GetFinished(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  Base* w;
+  ASSIGN_OR_RETURN_UNWRAP(&w, args.Holder());
+
+  char* buf = Malloc(EVP_MAX_MD_SIZE * 2);
+  size_t len = SSL_get_finished(w->ssl_, buf, EVP_MAX_MD_SIZE * 2);
+  if (!len)
+    return;
```

Comment:
This leaks memory, `buf` is not free’d here (the same applies below)

Reply:
Amended commit, now `free()`ing the `buf`fers in both functions. Thanks.